src/config.rs

```diff
@@ -1,54 +1,55 @@
+use config::ConfigError;
 use lazy_static::lazy_static;
+use serde::Deserialize;
+
 lazy_static! {
-    static ref CONF: Config = Config::default();
+    static ref CONF: Config = Config::new("config.toml".to_string()).unwrap();
```

Comment:
Ideally accept a command-line option to specify where the config file lives.

Parsing the file should be most definitely not done in a (lazy) static initialization, but should be done at the program startup and show some sensible error if something fails.

All that said, I think for this iteration this is fine, no need to change it now, but maybe create new issues for these topics.

Reply:
Yeah I also thought of taking filename from cmd-line but it requires changes at many places and like you mentioned I'll address it in different issue.